prime-router/docs/design/proposals/mapping-inventory-test-plan.md

```diff
@@ -0,0 +1,51 @@
+# Mapping Inventory Test Plan
+
+## Problem Statement
```

Comment:
This whole test plan is under problem statement?

Reply:
Ooops